### PR TITLE
Log webots.min.js modifications

### LINKED ISSUE #517: Change log for R2019b does not mention the webots.min.js feature
Ditto

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix #517

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: change_logs/ChangeLog.html [hunk 1]
```diff
@@ -23,6 +23,15 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
 </ul>
 <li><b>New Features</b></li>
 <ul>
+<li>Improved the Webots online 3D viewer: 'webots.min.js'</li>
+<ul>
+<li>Switched from 'X3DOM' to 'three.js'.</li>
+<li>Used ES6, Babel and minifyjs to provide 'webots.min.js' as a single and minified file.</li>
+<li>Decreased the rendering gap between Webots and 'webots.min.js'.</li>
```
Comment (omichel) [inline change_logs/ChangeLog.html]:
```suggestion
<li>Reduced the rendering difference between Webots and 'webots.min.js'.</li>
```

File: change_logs/ChangeLog.html [hunk 2]
```diff
@@ -23,6 +23,15 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
...
+<li>Added support of PBR appearances and of HDR textured background.</li>
```
Comment (omichel) [inline change_logs/ChangeLog.html]:
```suggestion
<li>Added support for PBR appearances and HDR textured background.</li>
```

File: change_logs/ChangeLog.html [hunk 3]
```diff
@@ -23,6 +23,15 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
...
+<li>Improved loading time and rendering rate.</li>
```
Comment (omichel) [inline change_logs/ChangeLog.html]:
```suggestion
<li>Improved loading time and rendering speed.</li>
```

File: change_logs/ChangeLog.html [hunk 4]
```diff
@@ -23,6 +23,15 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
...
+<li>Generalized the Webots websocket server to an HTTPServer serving the textures too.</li>
```
Comment (omichel) [inline change_logs/ChangeLog.html]:
```suggestion
<li>Generalized the Webots websocket server to an HTTP server serving the texture images too.</li>
```

**Comment by fabienrohrer** [comment]
Thank you for the requests; I applied them all as-is.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**